src/docker.ts

```diff
@@ -5,7 +5,11 @@ import { JAMSOCKET_CONFIG_DIR } from './jamsocket-config'
 
 export function getImagePlatform(imageName: string): { os: string, arch: string } {
   const getPlatform = spawnSync('docker', ['image', 'inspect', '--format', '{{.Os}} {{.Architecture}}', imageName])
-  const stdout = getPlatform.stdout.toString()
+  if (getPlatform.status !== 0) {
+    const stderr = getPlatform.stderr.toString().trim()
+    throw new Error(`Encountered docker error while checking image platform:\n${stderr}`)
```

Comment:
Maybe `while checking image platform` is too much of an implementation detail and we can just take it out?